google/cloud/bigquery/v2/minimal/internal/job_request.cc

```diff
@@ -139,7 +139,7 @@ StatusOr<rest_internal::RestRequest> BuildRestRequest(GetJobRequest const& r) {
   return request;
 }
 
-auto CastTimeToMilliseconds(const std::chrono::system_clock::time_point time) {
+auto CastTimeToMilliseconds(std::chrono::system_clock::time_point const time) {
   return std::chrono::time_point_cast<std::chrono::milliseconds>(time)
```

Comment:
Alternatively:

```
  return std::chrono::duration_cast<std::chrono::milliseconds>(time.time_since_epoch()).count();
```

In both cases this is assuming that `std::chrono::system_clock` epoch is the Unix Time epoch. This is not guaranteed in C++14 or C++17 (though in practice it is for any platform we care about). Consider a comment to at least indicate you considered this problem.